recipe/meta.yaml

```diff
@@ -118,7 +118,7 @@ outputs:
         - shimmy-atari
     test:
       imports:
-        - gymnasium.envs.atari
+        - echo "tested through test suite in gymnasium-all"
```

Comment:
Oh, sorry! My statement "Gymnasium's Atari environments are covered by the test suite." was wrong then. I must have mixed it up with shimmy's tests.

Reply:
> It is actually shimmy where we do the testing as the atari code doesn't exist in gymnasium but this isn't an issue

The question then becomes whether `gymnasium-atari` should be an output of the shimmy feedstock rather than this one? Not that I understand how the various components are intertwined but just thinking out loud.

Reply:
The fact that the inclusion of `shimmy-atari` in the test deps is causing a failure here actually does show that something about gymnasium-atari _is_ being covered here. So while we now have a test failure to fix (or skip), it does allay my concerns of where we should put this output (i.e. this feedstock is fine :)).